<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_client</class>
 <widget class="QDialog" name="form_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>إضافة عميل جديد</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ui.resources.qrc">
    <normaloff>:/icons/icons/users.png</normaloff>:/icons/icons/users.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>الصفة</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="error_mail">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_mail">
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="placeholderText">
        <string>personne@mail.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1" rowspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>معلومات أخرى</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="error_mobile">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_mobile">
       <property name="inputMask">
        <string>9999 99 99 99</string>
       </property>
       <property name="placeholderText">
        <string>9999 99 99 99</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>الجوال</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>العنوان</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="error_legal_status">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_legal_status">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>تحـديد الصفة ،،،</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2" rowspan="3" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="error_address">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_address"/>
     </item>
    </layout>
   </item>
   <item row="1" column="2" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="error_full_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_full_name">
       <property name="placeholderText">
        <string>الإسم و اللقب أو الهيئة المختصة ،،،</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>الإسم و اللقب</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>البريد الإلكتروني</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="5">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_cancel">
        <property name="text">
         <string>إلغاء</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_save">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>حفظ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="error_notes">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txe_notes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="error_phone">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_phone">
       <property name="inputMask">
        <string>999 99 99 99</string>
       </property>
       <property name="placeholderText">
        <string>xxx xx xx xx</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>الهاتف</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_legal_status</tabstop>
  <tabstop>le_full_name</tabstop>
  <tabstop>le_address</tabstop>
  <tabstop>le_phone</tabstop>
  <tabstop>le_mobile</tabstop>
  <tabstop>le_mail</tabstop>
  <tabstop>txe_notes</tabstop>
  <tabstop>pb_save</tabstop>
  <tabstop>pb_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../ui.resources.qrc"/>
 </resources>
 <connections/>
</ui>
